.npmignore

```diff
@@ -24,3 +24,5 @@ scripts/
 template-example/
 test/
 tsconfig.json
+example/.gitignore
+example/windows/.gitignore
```

Comment:
It looks like you're excluding `.gitignore` files here. Did you mean to add `!` in front?
```suggestion
!example/.gitignore
!example/windows/.gitignore

```

You should also add an newline to remove the warning:

![image](https://user-images.githubusercontent.com/4123478/124397101-22ea6780-dd0e-11eb-96f1-666d681ab47f.png)

Reply:
Yes, Sorry. I didn't know much about `.npmignore` and your feedback helped me. Thank you

Reply:
No worries. This file is a bit confusing. I forget to add things myself. 😄